### PR TITLE
Feature/randomize button

### PR DESCRIPTION
**Pull request by whiteheadol**
# Description

This PR adds basic styling for the homepage.

## Issues

Closes #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [x] Cosmetic changes (changes to layout or visuals in CSS file)

## How Has This Been Tested?

- [ ] console.log()
- [ ] dev tools
- [x] functionality check through web browser
- [ ] tested by viewing the markdown file preview
- [ ] tested with testing framework (Cypress)

## Checklist:

- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have deleted all comments/console.log() after functionality and understanding is final
- [x] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers

### CODE REVIEW

File: src/Components/Nav.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  font-size: 4em;
+}
+
+.btn-styling {
+  display: flex;
+  width: 50%;
+  justify-content: space-around;
+}
+
+button {
```
Comment (nbalmoori) [inline src/Components/Nav.css:18]: fancy button styling!

File: src/Components/Nav.js [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
+import { useSelector, useDispatch } from 'react-redux';
+import { getCurrentArt } from '../features/currentArt/currentArtSlice';
+import './Nav.css'
 
 const Nav = () => {
+
+  const dispatch = useDispatch();
+
+  const randomize = () => {
+    dispatch(getCurrentArt());
```
Comment (nbalmoori) [inline src/Components/Nav.js:11]: Concise way of getting this to work.

File: src/Components/Description.js [hunk 3]
```diff
@@ -1,16 +1,17 @@
...
   return (
-    <div>
-      <p className="art-title">Title: {currentArt.title}</p>
-      <p className="art-artist">Artist: {currentArt.artist}</p>
-      <p className="art-medium">Medium: {currentArt.medium}</p>
-      <p className="art-period">Period: {currentArt.century}</p>
+    <div className="image-description">
+      <p className="art-title"><strong>Title:</strong> {currentArt.title}</p>
+      <p className="art-artist"><strong>Artist: </strong>{currentArt.artist}</p>
+      <p className="art-medium"><strong>Medium: </strong>{currentArt.medium}</p>
```
Comment (Jlavarine) [inline src/Components/Description.js:13]: I think the bold looks really good!

**Review by Jlavarine — APPROVED** [review]
Great job!

**Pull request merged**